### PR TITLE
COMMERCE-6796 We need net shipping discount to properly calculate gro…

### PR DESCRIPTION
**Pull request by brianikim**
…ss shipping discount.

https://issues.liferay.com/browse/COMMERCE-6796

@riccardo-alberti

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/price/CommerceOrderPriceCalculationV2Impl.java [hunk 1]
```diff
@@ -210,6 +210,42 @@ public CommerceOrderPrice getCommerceOrderPrice(
 				shippingDiscountedWithTaxAmount =
 					shippingDiscountedWithTaxAmount.subtract(
 						discountAmountCommerceMoney.getPrice());
+
+				CommerceDiscountValue
+					orderShippingCommerceDiscountValueWithoutTax =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/price/CommerceOrderPriceCalculationV2Impl.java:215]: @brianikim we should name this 
```suggestion
					orderShippingWithoutTaxCommerceDiscountValue =
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by brianikim** [comment]
@dsitu , resolved the naming changes requested.

**Review by dsitu — APPROVED** [review]

**Comment by brianikim** [comment]
@riccardo-alberti ,

Let me know if this can be provided in a hot-fix.

**Comment by brianikim** [comment]
hey @dsitu ,

Resending a brand new PR here wth changes requested by @riccardo-alberti . We will only be altering tax value. Discount percentages display is working as intended:

**Pull request closed without merging**